charts/simcore-charts/common-helpers/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
```

Comment:
Q: in docker it is recommended to ignore everything, and then allow only the needed files. Is this here not also the case?

Reply:
this is autogenerated by helm